README.md

```diff
@@ -1 +1,5 @@
 # hello_world
+
```

Comment:
Provide some general information about the hello world doc before starting a new section